proposals/streams/README.md

```diff
@@ -0,0 +1,81 @@
+# MicroProfile Streams
+
+This proposal, in its current state, explores the question "what if Java offered a new API for handling streams of messages (either point to point or from a message broker) based on the JDK9 Flow API that was lighter weight and easier to use than JMS/MDBs".
+
+To start, we identify one very simple use case, we provide the absolute minimum API necessary to implement that use case, we implement that use case on top of that API, and then we provide an absolute minimum implementation of that API to demonstrate it in action. Subsequent iterations of this proposal can add to the API, but the aim for now is to provide a starting point for discussion and understanding.
+
+Note that the JDK9 Flow API of course requires a baseline requirement of JDK9 for MicroProfile. It may be found that by the time this spec is ready for inclusion in MicroProfile, that MicroProfile is not ready to increase its baseline JDK version to 9. In that case, the JDK9 Flow API can be replaced with the JDK8 compatible Reactive Streams API provided by http://www.reactive-streams.org, which is semantically identical to the JDK9 Flow API, the interfaces just live in a different package. So this proposal will talk about the JDK9 Flow API, as that's the long term target, but it may end up proposing something based on the JDK8 Reactive Streams interfaces. In the design of the API, this switch, and potential simultaneous compatibility for both by implementors, should be considered.
```

Comment:
I would suggest to just use JDK 8, so that this spec can be released sooner.

Reply:
@Emily-Jiang Would you mind elaborating on that?